src/utility.js

```diff
@@ -506,6 +506,27 @@ function setCustomItemKeys(item, custom) {
   }
 }
 
+function addErrorContext(item, errors) {
+  var custom = item.data.custom || {};
+  var contextAdded = false;
+
+  try {
+    for (var i = 0; i < errors.length; ++i) {
+      if (errors[i].hasOwnProperty('rollbarContext')) {
```

Comment:
is it safe to assume `errors[i]` is always an object? otherwise we might want a `typeof` check here before the `hasOwnProperty` just in case

Reply:
The code that calls this is already assuming a valid Error object, so should be safe to continue that assumption here since it isn't an external interface.